Sources/TuistKit/Services/Start/InitGeneratedProjectService.swift

```diff
@@ -0,0 +1,271 @@
+import FileSystem
+import Foundation
+import Mockable
+import Path
+import ServiceContextModule
+import TuistCore
+import TuistLoader
+import TuistScaffold
+import TuistSupport
+import XcodeGraph
+
+enum StartGeneratedProjectServiceError: LocalizedError, Equatable {
+    case ungettableProjectName(AbsolutePath)
+    case nonEmptyDirectory(AbsolutePath)
+    case templateNotFound(String)
+    case templateNotProvided
+    case attributeNotProvided(String)
+    case invalidValue(argument: String, error: String)
+
+    var type: ErrorType {
+        switch self {
+        case .ungettableProjectName, .nonEmptyDirectory, .templateNotFound, .templateNotProvided, .attributeNotProvided,
+             .invalidValue:
+            return .abort
+        }
+    }
+
+    var description: String {
```

Comment:
```suggestion
    var errorDescription: String? {
```

This is probably the reason why the error is not printed properly. It's quite easy to miss, so I wonder if we can add a linter of sorts that a `LocalizedError` _must_ have this property defined. Unfortunately, the `LocalizedError` protocol doesn't require it :/

Reply:
🤦🏼 We can implement a SwiftLint rule for this. I'll do that later.